db/seeds/index.js

```diff
@@ -1,10 +1,18 @@
 'use strict';
 
-const Movies = require('./data/movies');
+const Locations = require('./data/locations');
+const Movies    = require('./data/movies');
 
 exports.seed = function (Knex) {
```

Comment:
i get this lint error when i make this change. Not sure why.


/Users/vinay/Documents/tutorial/06_build_your_own_api/sfmovies/db/seeds/index.js
  6:32  error  "Promise" is already declared in the upper scope  no-shadow

✖ 1 problem (1 error, 0 warnings)